tests/inference/statements_test.py

```diff
@@ -1,6 +1,8 @@
+L1 = [5, 6, 7, 8]
+L2 = [9, 10, 7, 12]
+i = 0
+j = 0
 for x in [1, 2, 3]:
-    L1 = [5, 6, 7, 8]
```

Comment:
Why did you move this up? Also, WHY IS THERE NO TEST IN THIS PULL REQUEST THAT TESTS THE FUNCTIONALITY THAT WAS ADDED IN THIS PULL REQUEST! COME ON!